### PR TITLE
Adding pyspark example

### PR DESCRIPTION
**Pull request by murilommen**
## Description

Adding an example Jupyter Notebook to demonstrate how we've implemented our **experimental** pyspark integration. Feel free to read through the notebook and suggest anything that should be different.

### General Checklist

* [x] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [ ] Conform by the style guides, by using formatter
* [x] Documentation updated
* [ ] (optional) Please add a label to your PR

### CODE REVIEW

File: python/whylogs/api/pyspark/experimental/__init__.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from .profiler import (
     collect_column_profile_views,
+    collect_dataset_profile_view,
```
Comment (TheMellyBee) [inline python/whylogs/api/pyspark/experimental/__init__.py:3]: Good for now, but I'd like to think through the API names here. They are a bit long for the API although they are accurate. something like DatasetCollecter or something like that may be a easier to use, but still accurate?

**Review by TheMellyBee — APPROVED** [review]
This looks great. Couple things to check on:
1. Can this have an Open in Collab button?
2. Check to see if there is anyway to resolving the warning/errors. Especially the two blanks ones.

File: python/whylogs/api/pyspark/experimental/profiler.py [hunk 2]
```diff
@@ -56,3 +58,20 @@ def collect_column_profile_views(input_df: SparkDataFrame) -> Dict[str, ColumnPr
...
+) -> DatasetProfileView:
+    now = datetime.utcnow()
+
+    _dataset_timestamp = dataset_timestamp or now
+    _creation_timestamp = creation_timestamp or now
+
+    column_views_dict = collect_column_profile_views(input_df=input_df)
+
+    profile_view = DatasetProfileView(
+        columns=column_views_dict, dataset_timestamp=_dataset_timestamp, creation_timestamp=_creation_timestamp
```
Comment (jamie256) [inline python/whylogs/api/pyspark/experimental/profiler.py:74]: nice!

**Review by jamie256 — APPROVED** [review]
LGTM!

File: python/whylogs/api/pyspark/experimental/__init__.py (hunk 1 above)
Comment (jamie256) [reply python/whylogs/api/pyspark/experimental/__init__.py:3]: I'm with you on these being names we likely change, which is partly why these are in "experimental" namespace. I'm not against longer explicit naming for advanced scenarios in particular. Maybe we can get some input from users in our RC on this?

**Comment by jamie256** [comment]
> This looks great. Couple things to check on:
> 
> 1. Can this have an Open in Collab button?
> 2. Check to see if there is anyway to resolving the warning/errors. Especially the two blanks ones.

I also agree with Melly's suggestions, but we can do in a follow up too. The warnings I think Melly is referring to are the "red" cell output in the notebook example.

Also you might try this command to install whylogs[spark]:
```!pip install "whylogs[spark]" --pre```

**Comment by andyndang** [comment]
FYI you can't run Pyspark in Collab. They don't have Java

**Pull request merged**